backend/src/apiserver/server/util.go

```diff
@@ -287,12 +299,7 @@ func getUserIdentity(ctx context.Context) (string, error) {
 			return "", util.NewBadRequestError(errors.New("Request header error: unexpected number of user identity header. Expect 1 got "+strconv.Itoa(len(userIdentityHeader))),
 				"Request header error: unexpected number of user identity header. Expect 1 got "+strconv.Itoa(len(userIdentityHeader)))
 		}
-		userIdentityHeaderFields := strings.Split(userIdentityHeader[0], ":")
-		if len(userIdentityHeaderFields) != 2 {
-			return "", util.NewBadRequestError(errors.New("Request header error: user identity value is incorrectly formatted"),
-				"Request header error: user identity value is incorrectly formatted")
-		}
-		return userIdentityHeaderFields[1], nil
+		return getUserIdentityFromHeader(userIdentityHeader[0], common.GetKubeflowUserIDPrefix())
```

Comment:
Does GetKubeflowUserIDPrefix() return a prefix with ":" as the last character? Seems that the new logic doesn't require a ":" as the delimiter as the old logic does.

Reply:
Yes, you are right. We no longer require a ":" delimiter. This is expected behavior.